src/qt/rpcconsole.cpp

```diff
@@ -96,6 +96,16 @@ public Q_SLOTS:
 
 private:
     interfaces::Node& m_node;
+    static std::vector<std::string> parseHelper(const std::string& strCommand);
+    static bool commandMatches(const std::vector<std::string>& parsed_command, const std::string& key);
```

Comment:
Done. Since the compilar doesn´t actually allow what you suggest, i removed the declarations from the class and implemented this 2 method as independent functions.